src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,9 +2,6 @@
 
 ## Unreleased
 
-* Added `ILogger`/`Microsoft.Extensions.Logging` integration
```

Comment:
I've changed the wrong CHANGELOG file (it should be `OpenTelemetry` rather than `OpenTelemetry.Api`), now moving the change logs to the correct place.